File: snippets/csharp/BuyBtc.cs
using Breez.Sdk;

public class BuyBtcSnippets
{
  public void BuyBtc(BlockingBreezServices sdk)
  {
    // ANCHOR: buy-btc
    try
    {
      var buyBitcoinResponse = sdk.BuyBitcoin(
                      new BuyBitcoinRequest(BuyBitcoinProvider.MOONPAY));
    }
    catch (Exception)
    {
      // Handle error
    }
    // ANCHOR_END: buy-btc
  }
}

File: snippets/go/buy_btc.go
package example

import (
	"log"

	"github.com/breez/breez-sdk-go/breez_sdk"
)

func BuyBitcoin() error {
	// ANCHOR: buy-btc
	buyBitcoinRequest := breez_sdk.BuyBitcoinRequest{
		Provider: breez_sdk.BuyBitcoinProviderMoonpay,
	}
	buyBitcoinResponse, err := sdk.BuyBitcoin(buyBitcoinRequest)
	if err != nil {
		return err
	}
	log.Printf("%#v", buyBitcoinResponse)
	// ANCHOR_END: buy-btc
	return nil
}
